man/man8/zpool.8

```diff
@@ -1926,8 +1926,10 @@ result in partially resilvered devices unless a second scrub is performed.
 .Ar pool Ar device Ns ...
 .Xc
 Removes the specified device from the pool.
-This command currently only supports removing hot spares, cache, log
-devices and mirrored top-level vdevs (mirror of leaf devices); but not raidz.
+This command supports removing hot spare, cache, log, and both mirrored and
+non-redundant primary top-level vdevs, this includes dedup and special vdevs.
+When the primary pool storage includes a top-level raidz vdev only hot spare,
+cache, and log devices can be removed.
```

Comment:
I get this, but IMO it would be more easily understood to say `When the primary pool storage includes a raidz vdev, other top-level vdevs may not be removed.`

Reply:
I considered a wording very similar, but decided against it in favor of this clunkier more precise wording.  My concern was that it implied spare, cache, and log devices, which look like top-level vdevs in the `zpool status` output, could not be removed.  But I'd love a better way to phase this.